README.md

```diff
@@ -102,7 +102,7 @@ Options:
 
 A protocol type for rendering images of commit graphs.
 By default `auto` will guess the best supported protocal for the current terminal.
-Kitty, Ghostty, and Konsole terminals are detected as `kitty` via environment variables, and all others are assumed to support `iterm`.
+The Kitty and Ghostty terminals are detected as `kitty` via environment variables, and all others are assumed to support `iterm`.
```

Comment:
Please see my issue commend, Konsole can be added back.

Reply:
https://github.com/lusingander/serie/pull/24#issuecomment-2265333962